nightingale_msgs/srv/RobotConfigurationLookup.srv

```diff
@@ -5,5 +5,8 @@ int8 RIGHT_ARM=1
 int8 TORSO=2
 int8 HEAD=3
 int8 body_part
+bool lookup_cart
+string joint_name
```

Comment:
all cartesian poses are from base_link -> body_part end effector, so I believe the joint name can be removed.